Introductory Tests/traj_opt.ipynb

```diff
@@ -0,0 +1,261 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    import math
</pre><p>I don't think you use this import.</p><p> </p><p>I would move all imports to the top of the notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rasmusmarak/67P-Swarm/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>